### DISCUSSION TITLE
Flipper Zero Resources

### DISCUSSION THREAD

**Discussion by cpoore1** [ainfosec/FISSURE#38] [General]
I recently got a Flipper Zero and I am now able to test it out. I'm combing through the GitHub pages and I would like to know about your favorite Flipper Zero resources. What do you think should be added to the FISSURE installer, menu items (tools/links), scripts, or as part of a Flipper Zero lesson for people just getting started?

**Comment by xxanxnie** [comment]
**[Awesome Flipper](https://github.com/djsime1/awesome-flipperzero)** contains a ton of collections with resources  for **[Flipper Zero](https://flipperzero.one/)** device. 

Below are some that could be applied to FISSURE:

1. **[flipperzero-bruteforce](https://github.com/tobiabocchi/flipperzero-bruteforce)**: Able to generate brute force .sub force files for Sub-GHz protocols that use fixed OOK codes. Sub-GHz is a protocol that operates in a frequency band less than 1GHz, usually around MHz. This is an essential technologies for wireless communication. 

2. **[Flipper_Zero-BadUsb](https://github.com/UNC0V3R3D/Flipper_Zero-BadUsb)**: A collections of BadUSB scripts for Flipper Zero that might be used for lesson. Has a bunch of explanation on usage for newbies on using the scripts. However, seems to only work on WIndow.

3. [**fztea**](https://github.com/jon4hz/fztea): Basically a remote control flipper from local terminal or remote over SSH which could be cool to have running. Can be maybe added to installer?

4. **[create_sub.py](https://gist.github.com/jinschoi/f39dbd82e4e3d99d32ab6a9b8dfc2f55)**: Another python script to generate Flipper RAW .sub from OOK bitstreams. Could be useful during FISSURE signal analysis with Flipper Zero. 

5. [**flipperzero-nrf24**](https://github.com/mothball187/flipperzero-nrf24): An  NRF24 driver for the Flipper Zero device. The NRF24 is a popular line of 2.4GHz radio transceivers from Nordic Semiconductors.

6. **[flipper_toolbox](https://github.com/evilpete/flipper_toolbox)**: Contains random scripts and links for generating/creating/modifying Flipper Zero data files that can be loaded onto a Flipper. All tools are written in python. Some tools includes reading Flipper NFC dump and add ascii and hex to RFID Hex dump, decoding subGhz, read SubGhz RAW file and plot using pyplot. Definitely should check out.

7. **[subghz_ook_to_sub.py](https://github.com/evilpete/flipper_toolbox/blob/main/subghz_ook_to_sub.py)**: Found in the above repo. Convers signal captured with rtl_443 or rtl_sdr to FlipperZero .sub format. FISSURE integrates rtl_sdr and this could be useful tool for conversion.
